ocaml/testsuite/tests/backtrace/names.ml

```diff
@@ -97,6 +102,16 @@ let inline_object f =
   end in
   obj#meth
 
+let[@inline never] labelled_arguments_partial k =
+  let f ~a = ignore a; k (); fun ~b -> ignore b; () in
```

Comment:
It looks like this `f` is getting inlined, ruining the effect. Try adding a noinline or opaque_identity on `f`.

Reply:
This doesn't seem to have worked either.  I am still pondering why these partial ones don't seem to work (also see below).

Reply:
Ah, the problem is that the location needs to be passed around the loop of `build_apply` I think.  I'm trying to fix this.